eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/steps/processors/VariantProcessorTest.java

```diff
@@ -0,0 +1,35 @@
+package uk.ac.ebi.eva.accession.pipeline.steps.processors;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import uk.ac.ebi.eva.accession.core.SubmittedVariant;
+import uk.ac.ebi.eva.commons.core.models.IVariant;
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+
+import static org.junit.Assert.assertEquals;
+
+public class VariantProcessorTest {
```

Comment:
Again when reformatted using our formatting template, line length should change in this class.